### PR TITLE
[zh] Fix small typo in Namespaces documentation

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/45763/files

### CODE REVIEW

File: content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ---
 title: 名字空间
+api_metadata:
+  - apiVersion: "v1"
+    kind: "Namespace"
```
Comment (dipesh-rawat) [inline content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md:5]: In the front-matter there seems to be some extra spaces compared to the English version of the file.

```suggestion
api_metadata:
- apiVersion: "v1"
  kind: "Namespace"
```

Comment (Arhell) [reply content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md]: Done, thanks

File: content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md [hunk 2]
```diff
@@ -9,20 +12,23 @@ reviewers:
...
 <!--
-In Kubernetes, _namespaces_ provides a mechanism for isolating groups of resources within a single cluster. Names of resources need to be unique within a namespace, but not across namespaces. Namespace-based scoping is applicable only for namespaced {{< glossary_tooltip text="objects" term_id="object" >}} _(e.g. Deployments, Services, etc)_ and not for cluster-wide objects _(e.g. StorageClass, Nodes, PersistentVolumes, etc)_.
+In Kubernetes, _namespaces_ provide a mechanism for isolating groups of resources within a single cluster. Names of resources need to be unique within a namespace, but not across namespaces. Namespace-based scoping is applicable only for namespaced {{< glossary_tooltip text="objects" term_id="object" >}} _(e.g. Deployments, Services, etc)_ and not for cluster-wide objects _(e.g. StorageClass, Nodes, PersistentVolumes, etc.)_.
 -->
 在 Kubernetes 中，**名字空间（Namespace）** 提供一种机制，将同一集群中的资源划分为相互隔离的组。
 同一名字空间内的资源名称要唯一，但跨名字空间时没有这个要求。
 名字空间作用域仅针对带有名字空间的{{< glossary_tooltip text="对象" term_id="object" >}}，
 （例如 Deployment、Service 等），这种作用域对集群范围的对象
-（例如 StorageClass、Node、PersistentVolume 等）不适用。
+（例如 StorageClass、Node、PersistentVolume 等。）不适用。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md:31]:
```suggestion
（例如 StorageClass、Node、PersistentVolume 等）不适用。
```

Comment (Arhell) [reply content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md]: Done

**Pull request merged**